guides/mn-application-aws-lambda-java11/mn-application-aws-lambda-java11.adoc

```diff
@@ -6,29 +6,16 @@ common:completesolution.adoc[]
 
 common:create-app-features.adoc[]
 
-The generated application contains a `BookController`. It responds to POST requests to `/`.
+The generated application contains a `HomeController`. It responds to POST requests to `/`.
 
-source:BookController[]
+source:HomeController[]
 
 callout:controller[arg0=/]
-* The `@Post` annotation maps HTTP requests to `/` to the `save` method.
-* Add the `@Valid` annotation to any method parameter which requires validation.
-
-The controller method parameter is a `Book` object:
-
-source:Book[]
-
-* Annotate the class with `@Introspected` to generate the Bean Metainformation at compile time.
-
-It returns a `BookSaved` object:
-
-source:BookSaved[]
-
-* Annotate the class with `@Introspected` to generate the Bean Metainformation at compile time.
+* The `@Get` annotation maps HTTP requests to `/` to the `index` method.
```

Comment:
maybe use callout:get?

Reply:
I think that we don't support callouts without `number` yet.